index.html

```diff
@@ -30,16 +58,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
```

Comment:
for each both you can do multiple things. You can use multiple classes `<div class="box box1">`, `<div class="box" id="box1">` or use css selectors :nth-child() and just keep the original way `<div class="box">`